cli/pkg/kctrl/cmd/core/semver_value.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 20210 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package core
+
+import (
+	uitable "github.com/cppforlife/go-cli-ui/ui/table"
+	versions "github.com/vmware-tanzu/carvel-vendir/pkg/vendir/versions"
+)
+
+type ValueSemver struct {
+	V uitable.Value
+}
+
+func NewValueSemver(v uitable.Value) ValueSemver {
```

Comment:
seems like taking a string might remove extra indirection.

Reply:
Yep, will do that.

Reply:
Done.